### ISSUE TITLE
Spare Pin

### ISSUE THREAD

**Issue by serrq** [horizontalsystems/unstoppable-wallet-android#6791]
Situation: 
Currently I have 8 wallets all reachable by my master pin. One of these is reachable by duress pin while the other ones are kept hidden.

Problem:
I want launch at the opening only selected wallets (I am not under coercion action) to avoid showing to my colleagues or just to prying eyes near me (train, bus, etc.) what are my entire assets.

Solution:
Just an other duress pin for not coercion situations. It allows me to choise which wallets showing when I type this pin.

In summary:
I just want to hide my one million dollars wallet (so to speak) while I have funds to spend on daily expenses.

How we want naming this pin?
Secondary pin? Daily pin? Spare pin? Standard mode?

**Comment by abdrasulov** [comment]
You can achieve it by setting the next level Duress pin inside of the Duress pin. The duress pin mode works exactly like the master pin. For instance, there are 3 wallets, named A, B, and C. The master pin has access to all of them. The first Duress pin has access to A and B. The second Duress pin has access to only A.

**Comment by abdrasulov** [comment]
Yes. There can be any number of duress pins.

**Comment by abdrasulov** [comment]
Exactly. The main idea is to make the duress mode work just like the master pin mode. So nobody can guess that the current mode is duress or master.
